### PR TITLE
Update sig-scale charter

### PR DESCRIPTION
**Pull request by rthallisey**
**What this PR does / why we need it**:
Update Sig-Scale charter

```release-note
NONE
```

### CODE REVIEW

**Review by alaypatel07 — APPROVED** [review]
/lgtm 

Thanks @rthallisey

**Comment by fabiand** [comment]
Is an update ot https://github.com/kubevirt/community/blob/main/sigs.yaml needed?

**Comment by fabiand** [comment]
@rthallisey how can we conervege aka merge tihs?

**Comment by rthallisey** [comment]
@fabiand yes, added.

File: sigs.yaml [hunk 1]
```diff
@@ -81,11 +81,17 @@ sigs:
...
+            - https://raw.githubusercontent.com/kubevirt/kubevirt/main/tools/OWNERS
+            - https://raw.githubusercontent.com/kubevirt/kubevirt/main/hack/OWNERS
+            - https://raw.githubusercontent.com/kubevirt/kubevirt/main/tests/performance/OWNERS
       label: kubevirt-community
       leadership:
         chairs:
           - github: rthallisey
-            name: "Ryan T. Hallisey"
+            name: "Ryan Hallisey"
             company: Nvidia
```
Comment (aburdenthehand) [inline sigs.yaml:95]:
```suggestion
            company: Nvidia
            name: Alay Patel
            company: Nvidia
```

**Pull request merged**